src/app/views/sidebar/resource-explorer/ResourceLink.tsx

```diff
@@ -27,6 +28,16 @@ const ResourceLink = (props: IResourceLinkProps) => {
   const paths = collections?.find(k => k.isDefault)?.paths || [];
   const resourceLink = { ...link };
 
+  const [alwaysShowButtons, setAlwaysShowButtons] = useState(false);
+
+  useEffect(() => {
+    variantService.getFeatureVariables('default', 'alwaysShowButtons').then((value) => {
```

Comment:
If the known variants could be stored in a constants file, it would be easy to add / remove / change them and reference them in the code where they are used

Reply:
hmm you are saying we should have type variants? where all variants will be stored? or we should be saving all variants as constants in the constants file?

Reply:
All variants in a constants file

Reply:
Oh got it. This is a great idea, it would help us know what variants have been declared